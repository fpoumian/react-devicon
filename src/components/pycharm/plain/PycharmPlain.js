
import React from 'react'
import PropTypes from 'prop-types'
import SVGInline from 'react-svg-inline'
import iconSVG from './PycharmPlain.svg'
import { normalizeUnit } from '../../utils.js'

/** PycharmPlain */
function PycharmPlain({width, height}) {
  return (
     <SVGInline
      classSuffix="-devicon"
      className="PycharmPlain"
      svg={iconSVG}
      width={normalizeUnit(width)}
      height={normalizeUnit(height)}
    />
  )
}

PycharmPlain.propTypes = {
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
}

export default PycharmPlain
