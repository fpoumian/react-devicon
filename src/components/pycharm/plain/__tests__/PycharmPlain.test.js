import React from 'react'
import renderer from 'react-test-renderer'
import PycharmPlain from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<PycharmPlain />).toJSON()
  expect(tree).toMatchSnapshot()
})
