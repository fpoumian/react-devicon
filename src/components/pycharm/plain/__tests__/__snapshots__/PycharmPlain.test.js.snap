// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="SVGInline PycharmPlain"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon PycharmPlain-devicon\\" id=\\"pycharm\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 128 128\\"><defs><style>.cls-1{fill:#4d8548}</style></defs><title>Artboard 15</title><g id=\\"plain\\"><path class=\\"cls-1\\" d=\\"M96.44 40.53h-.52a23.59 23.59 0 0 0-23 17.68h-.12a20.22 20.22 0 0 1-40.08-3.8 16 16 0 0 1 .16-2.23c0-.15 0-.3.06-.45s.1-.57.16-.85A64 64 0 0 1 66.77 7.09 49.23 49.23 0 0 0 3.86 50.15c-.06.18-.15.35-.21.54v72h29.16v-23.3a49.24 49.24 0 0 0 67.45-58.86h-3.82z\\"/><path class=\\"cls-1\\" d=\\"M67.95 57.28l.09-.15a28.4 28.4 0 0 1 27.54-21.55h13.94a15.1 15.1 0 0 0 6.19-1.58c-.48-.06-1-.13-1.45-.23a19.93 19.93 0 0 1-15.12-13.44 20 20 0 0 0 11.94 7.45A19.76 19.76 0 0 0 123 26.42a15.22 15.22 0 0 0-13.88-21.28L98 5.11h-2.4a58.89 58.89 0 0 0-57.53 46.33 15 15 0 0 0-.39 3 15.27 15.27 0 0 0 30.27 2.87zm36.45-43.36a4.73 4.73 0 1 1 3.72 5.56 4.74 4.74 0 0 1-3.73-5.55zm20 93.73a15.24 15.24 0 0 1-15.24 15.24H95.58a58.57 58.57 0 0 1-38.52-14.42 54 54 0 0 0 34.82-16.32 29.37 29.37 0 0 0 3.7.26h13.53a15.25 15.25 0 0 1 15.24 15.25z\\"/></g></svg>",
    }
  }
/>
`;
